spec/helpers.rb

```diff
@@ -122,12 +122,14 @@ def make_instance instance_data = {}
 module PlatformHelpers
   DEBIANPLATFORMS = %w[debian ubuntu]
 
-  SYSTEMDPLATFORMS = %w[amazon-2023
-                        fedora
-                        el-
-                        centos
-                        redhat
-                        oracle
-                        scientific
-                        archlinux].concat(DEBIANPLATFORMS)
+  RHELPLATFORMS = %w[
```

Comment:
I'd say REDHATPLATFORMS is better because Fedora and Amazon 2023 aren't RHEL. Amazon 2023 (and beyond) is based on Fedora, just like RHEL is based on Fedora.